README.md

```diff
@@ -6,8 +6,8 @@
 
 ----
 
-![Build](https://github.com/pymc-labs/pymc-marketing/actions/workflows/ci.yml/badge.svg)
-![Test](https://github.com/pymc-labs/pymc-marketing/actions/workflows/test.yml/badge.svg)
+![Build](https://github.com/pymc-labs/pymc-marketing/actions/workflows/test.yml/badge.svg)
+![Test](https://github.com/pymc-labs/pymc-marketing/actions/workflows/test_notebook.yml/badge.svg)
```

Comment:
Is there a doc build badge for "Build"

Reply:
You mean like https://github.com/pymc-labs/pymc-marketing/pull/1278/commits/3488cc461ec6ac9fbcacdac3add7f06a8813b90b ?